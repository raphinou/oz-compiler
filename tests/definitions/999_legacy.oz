% This code failed in an ealier version, left here for safety.
local
   A P B
in
   proc {P V}
      T
   in
      proc {T U}
         Text=44
      in
         {Show Text}
         {Show U}
         {Show A}
      end
      {T V}
   end
   A = 5
   B = 7
   {P A}
   {For 1 5 1 P}
end
