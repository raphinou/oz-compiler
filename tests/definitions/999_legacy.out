44
5
5
44
1
5
44
2
5
44
3
5
44
4
5
44
5
5
